versioned_docs/version-0.x.0/kubeslice-cli/troubleshooting-guide.mdx

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting Guide
+
+This guide describes troubleshooting scenarios that you could face during installing and while using the `kubeSlice-cli` tool.
+
+### Unable to Install Kubeslice using the kubeslice-cli Tool on Ubuntu
+
+During the installation of KubeSlice using the `kubeslice-cli install -p=minimal-demo` command, if you get the following error message: 
+
+```
+✓ Writing configuration 📜
+ • Starting control-plane 🕹️  ...
+ ✗ Starting control-plane 🕹️
+ERROR: failed to create cluster: failed to init node with kubeadm: command "docker exec --privileged ks-w-2-control-plane kubeadm init --skip-phases=preflight --config=/kind/kubeadm.conf --skip-token-print --v=6" failed with error: exit status 137
+
+Command Output:
+2022/10/04 06:12:21 Process failed exit status 1
+```
+**To resolve**:
+
+There could be a memory/disk space issue. 
+- Remove unused clusters (other than the ones used in the demo).
+- Increase disk space/memory resources.
+
+### Unable to run the kubeslice-cli commands
+After successful installation of KubeSlice using kubeslice-cli, you are unable to use the commands:
+
+```
+kubeslice-cli get sliceConfig -n kubeslice-demo
+```
+```
+Fetching KubeSlice sliceConfig...
+🏃 Running command: /usr/local/bin/kubectl get sliceconfigs.controller.kubeslice.io -n demo
+error: the server doesn't have a resource type "sliceconfigs"
+2022/10/04 08:26:40 Process failed exit status 1
+```
+**To resolve**:
+- Ensure you are on the controller cluster to run the commands: `kubectx -c`.
```

Comment:
If they should be performed in an order, then number them in the list.

Reply:
no order required here.